modules/ROOT/pages/api-analytics-dashboard.adoc

```diff
@@ -10,14 +17,7 @@ For Gold and Platinum subscriptions to Anypoint Monitoring, API analytics data a
 . Sign into Anypoint Platform and navigate to *Anypoint Monitoring*.
 . Select the Environment, Resource name, and Version/Instance for the API to view.
 . Click *View*. +
-The API analytics page is displayed, where you can view information about a running, managed API, including: +
-* Requests
-* Requests by HTTP response code
-* Requests by method
-* Requests (Avg), Average Size of Requests, and Average Size of Responses
-* Requests by HTTP policy violation
-* Requests by failure
-+
+The API analytics page is displayed, where you can view information about a running, managed API. +
```

Comment:
Can you make it active?

Reply:
The screenshot guidelines say you need a caption. I think.

Reply:
```suggestion
On the API analytics page, you can view information about a running, managed API. +
```